evm_arithmetization/src/cpu/kernel/tests/transaction_parsing/parse_type_0_txn.rs

```diff
@@ -8,6 +8,7 @@ use crate::cpu::kernel::aggregator::KERNEL;
 use crate::cpu::kernel::constants::txn_fields::NormalizedTxnField;
 use crate::cpu::kernel::interpreter::Interpreter;
 
+#[ignore]
```

Comment:
So weirdly, it seems to fail with something unrelated to your changes but from this TODO in `type_0_computed_signed_data`:

```asm
    %alloc_rlp_block POP // Doesn't work otherwise. TODO: Figure out why.
    %alloc_rlp_block
```

Removing the dummy first line solves this. I can't recall the context when it was added, nor if it is still an issue (or rather a symptom of a deeper issue in the kernel), but it seems enough for this test to pass.

Reply:
Actually it seems it's been already removed from `develop` and all `feat/` branches, so maybe just a `git merge` leftover

Reply:
Actually I just recalled that this got added in the first implem of type2 on all 3 types of transactions, so not sure removing this dummy line for now is a good solution long term.

Reply:
Hm actually it seems like all tests are passing again without this `%alloc_rlp_block POP`, so I've removed it. Don't really know what happened there.

Reply:
For type_0, but it's failing for the other types (like `erc20` test for type2)

Reply:
> For type_0, but it's failing for the other types (like `erc20` test for type2)

Hm I think it's passing on this branch

Reply:
If we remove this dummy line in the type_2 ASM file it panics:

```asm
type_2_compute_signed_data:
    // %alloc_rlp_block POP // Doesn't work otherwise. TODO: Figure out why.
    %alloc_rlp_block
```

Reply:
(because `erc20` test uses a type-2 txn)

Reply:
Ah yeah right

Reply:
The `process_type_0_txn` does seem to pass with the `%alloc_rlp_block POP` though.

Reply:
Yeah this one is fine. I commented back saying nevermind after seeing this wasn't still solved for type1 / type2.